src/core_atmosphere/physics/mpas_atmphys_init.F

```diff
@@ -408,6 +431,29 @@ subroutine physics_init(dminfo,clock,configs,mesh,diag,tend,state,time_lev,diag_
     endif
  endif
 
+!initialization of non-stationary gravity wave drag
+ if(trim(gwdo_scheme).eq.'bl_ugwp_gwdo') then
+    ! Read in ugwp_oro_data
+    call MPAS_stream_mgr_read(stream_manager, streamID='ugwp_oro_data_in', whence=MPAS_STREAM_NEAREST, ierr=ierr)
```

Comment:
Is there a reason why this stream isn't being read with the other streams in `mpas_atm_core.F`?

Reply:
Yes.  Reading in this stream (as well as ‘ugwp_ngw_in’) at this point in the code avoids the need to have the time variable ‘xtime’ included in the files.  Otherwise, the files are read in at the beginning of the first time step and the model terminates if there is no ‘xtime’ variable in the files read in at the point.

Reply:
@mdtoyNOAA I was looking at Michael Duda's emails, and he thinks that you should only get a warning without xtime being defined. I'm trying to recreate your code to read  neural networks into MPAS, and I think I can read streams from mpas_atm_core.F without needing an xtime variable. I do get a warning but not a model crash.

Reply:
I still got the warnings and model crashes with the stream_mgr_read call in mpas_atm_core.F, even after M. Duda's suggestion.  So I went with his suggestion of using stream_mgr_read followed by a call to stream_mgr_reset_alarms in mpas_atmphys_init.F, and that works.  I do get the warning message, however, but not  a crash.

Reply:
I will look into moving the call to the ugwp_ngw_in read stream to mpas_atm_core.F in a follow-up PR.